frame/assets/src/functions.rs

```diff
@@ -55,7 +55,12 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 	/// Get the total supply of an asset `id`.
 	pub fn total_supply(id: T::AssetId) -> T::Balance {
-		Asset::<T, I>::get(id).map(|x| x.supply).unwrap_or_else(Zero::zero)
+		Self::maybe_total_supply(id).unwrap_or_else(Zero::zero)
```

Comment:
Just wondering: why was it not `unwrap_or_default` before?

Reply:
But also fine this way.